dbms/src/Storages/Page/V3/Universal/UniversalPageIdFormatImpl.h

```diff
@@ -109,6 +109,18 @@ struct UniversalPageIdFormat
         return buff.releaseStr();
     }
 
+    // RAFT_PREFIX LOCAL_PREFIX REGION_RAFT_PREFIX region_id APPLY_STATE_SUFFIX
+    static UniversalPageId toRaftApplyStateKeyInRaftEngine(UInt64 region_id)
```

Comment:
What's this?

Reply:
We use it to resolve RaftApplyState when TiFlash-proxy transfer its data from KVEngine to RaftEngine later (for example, adapting to RaftStore v2)

https://github.com/pingcap/tiflash/pull/7858/files#diff-993429dba2f708aefb20bb3620fa3d03dd72561a21bf370eb708a1821b47ce1fR33-R42